angular/_blueTheme.scss

```diff
@@ -330,6 +330,9 @@
         }
         .pxb-info-list-item-content {
             color: map-get($pxb-gray, 500);
```

Comment:
Should the icon be secondary-text here, or is gray[500] preferred?

Reply:
React is using text secondary for the icon as the ILI default